wiki/Linux-Troubleshooting.md

```diff
@@ -11,34 +11,47 @@ This path might differ based on your Steam installation, in that case SteamVR wi
 Then pick the storage location with the star emoji (⭐) and take the path directly above the usage statistics. Prepend this path to `steamapps/common/SteamVR/bin/vrmonitor.sh`.
 Finally put this entire path into the SteamVR commandline options instead of the other one.
 
-### Hyprland/Sway/Wlroots Fix
+### Hyprland/Sway/Wlroots Qt fix
 
 If you're on hyprland, sway, or other wlroots-based wayland compositor, you might have to prepend `QT_QPA_PLATFORM=xcb` before commandline, which results in full commandline for steamvr being something like this:
 `QT_QPA_PLATFORM=xcb ~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command%`.
 
 Related issue:
 [[BUG] No SteamVR UI on wlroots-based wayland compositors (sway, hyprland, ...) with workaround](https://github.com/ValveSoftware/SteamVR-for-Linux/issues/637).
 
+
+## The alvr driver doesn't get detected by SteamVR (even after vrmonitor fix)
+
+Could be related to Arch AUR package (either installed not for nvidia on nvidia based system (`alvr-nvidia`), or just in general).
+
+### Fix
+
+Try using a launcher or portable .tar.gz release from the Releases page.
+
 ## Artifacting, no SteamVR Overlay or graphical glitches in streaming view
 
-Could be related to AMD amdvlk driver being present on your system.
+Could be related to AMD amdvlk or amdgpu-pro driver being present on your system.
 
 If you have Amdvlk installed on your system, it overrides other vulkan drivers and causes SteamVR to break. Use the `vulkan-radeon` driver (aka radv) instead.
 
 ### Fix
 
-Check if Amdvlk is installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall Amdvlk from your system.
+Check if amdvlk or other vulkan drivers are installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall amdvlk from your system.
+
+On arch, first install `vulkan-radeon` and uninstall other drivers.
```

Comment:
would be nice to also include how to check for amdgpu-pro since we should only need to extend the grep

Reply:
From what I can tell, its starting with same name (amd_icd), but not sure If its in the same place (might be in /opt, but idk how vulkan behaves in this case, how does it know where to look??)

Reply:
https://wiki.gentoo.org/wiki/AMDGPU-PRO
This lists them for gentoo, no clue for arch. But tbh I actually think we can ignore that in the detection because you must do some pretty cursed shit to install amdgpu-pro. And in that case the user can go figure it out themselves
But fwiw, I think that path always needs to be one or rather it's always one centralized path, even if it might vary between distros

Reply:
arch too seems to install it as amd_pro `mv "${pkgdir}"/opt/amdgpu-pro/etc/vulkan/icd.d/amd_icd64.json "${pkgdir}"/usr/share/vulkan/icd.d/amd_pro_icd64.json`.
For ubuntu (that what this aur script is repacking it seems to be a much bigger mess however), but since arch is like 80% of linux gaming I think just checking for amd_pro too should be enough